nvsvocprez/utilities/config.py

```diff
@@ -0,0 +1,30 @@
+"""Reads .env file and sends the config to authentication processes."""
+from starlette.config import Config
+
+
+class MissingKeyError(Exception):
+    """Raised when the env file is missing one or more keys."""
+
+
+expected_config_keys = {
```

Comment:
login secrets are "REQUIRED" but only if login is enabled. Modify conditionals accordingly.